pkg/handlers/authorizer/decisiontree.go

```diff
@@ -8,9 +8,9 @@ import (
 )
 
 type DecisionTreeCmd struct {
-	PolicyID  string `name:"policy_id" required:"" help:"policy id"`
-	Path      string
-	Decisions []string
+	AuthParams `embed:""`
+	Path       string   `name:"path" help:"policy package to evaluate"`
+	Decisions  []string `name:"decisions" help:"policy decisions to return"`
```

Comment:
@gertd should Decision be required? If left optional perhaps it should default to `["*"]` to return all decisions.

Reply:
The default was that it evaluates everything underneath the path, unless restricted, which I believe follows the implementation in the authorizer when calling decision tree. Bottomline I would think it make sense for the default between the CLI and authorizer API to align.

Reply:
Updated the default to `"*"` so that all decisions are evaluated unless stated otherwise.
Without it we return empty results and it may not be immediately obvious why.